app/src/main/assets/defaults/DefaultTiles.json

```diff
@@ -0,0 +1,74 @@
+{
+  "default_tiles": [
+    {
+      "tile_0": {
+        "url": "https://youtube.com/tv",
+        "title": "YouTube",
```

Comment:
Thinking a little more about the structure of this json object, we should have an "identifier" field, with a string that describes the title but without spaces (like "youtube" or "hollywoodReporter").